apps/api-users/src/schema/user/user.spec.ts

```diff
@@ -200,7 +200,7 @@ describe('api-users', () => {
       })
       expect(prismaMock.user.findUnique).toHaveBeenCalledWith({
         where: {
-          id: '1'
+          userId: user.id
```

Comment:
shouldn't this be 
```suggestion
          userId: user.userId
```

but also can you edit the underlying mock so that user id and user userId aren't 1?